frontend/src/components/Utils/time.js

```diff
@@ -78,4 +76,32 @@ export function timezonesForCountry(countryName) {
   return zones.map(e => {
     return e.replace(/[_]/g, ' ');
   });
+}
+
+/**
+ * Get a date in 'YYYY-MM-DD' form. 
+ * 
+ * @param {string} msTimestamp Timestamp, in milliseconds since epoch.
+ * @param {string} timezone The timezone which the string should be returned in.
+ * @returns {string} The date in 'YYYY-MM-DD' form. 
+ */
+export function getDateBarebones(msTimestamp, timezone=null) {
```

Comment:
Nit (optional): I believe this is the ISO date format so a more clean name for this function may be something like `getISODateString`.